examples/breakout/breakout_stdp_right.py

```diff
@@ -0,0 +1,56 @@
+import torch
```

Comment:
What's the purpose of this script? We want to have only a few examples in the BindsNET repo so we don't have to maintain a bunch.

Reply:
@Hananel-Hazan wanted an experiment for reinforcing only the right action via STDP. This was to see if it performs well enough to include the results in the paper. At this point, I am not really sure which one we should use. The breakout.py example performs the best. Which is not saying much because it has no learning.

Reply:
Got it, thanks. It might be better to keep this script in the [bindsnet_experiments repo](https://github.com/djsaunde/bindsnet_experiments).

Reply:
Ok. I will remove it for now. Since I am not sure about the utility of this experiment.

Reply:
It should be in directory with the rest of the benchmark scripts and example scripts that will appear in the paper for Frontier. @djsaunde will create a spacial tag for it